editor/components/rich-text/tinymce.js

```diff
@@ -96,6 +182,23 @@ export default class TinyMCE extends Component {
 		} );
 	}
 
+	saveEditorNode( editorNode ) {
+		this.editorNode = editorNode;
+
+		/**
+		 * A ref function can be used for cleanup because React calls it with
+		 * `null` when unmounting.
+		 */
+		if ( this.removeInternetExplorerInputFix ) {
+			this.removeInternetExplorerInputFix();
+		}
+
+		this.removeInternetExplorerInputFix =
+			editorNode && needsInternetExplorerInputFix( editorNode ) ?
+				applyInternetExplorerInputFix( editorNode ) :
+				null;
```

Comment:
Minor: I think this would read better as an `if`, e.g.

```js
if ( editorNode && needsInternetExplorerInputFix( editorNode ) ) {
	this.removeInternetExplorerInputFix = applyInternetExplorerInputfix( editorNode );
}
```